database/buildSchema/29_review_response.sql

```diff
@@ -1,9 +1,11 @@
 -- review response
 
-CREATE TYPE public.review_response_decision as ENUM ('Approve', 'Decline');
+CREATE TYPE public.review_response_decision as ENUM ('Approve', 'Decline', 'Agree', 'Disagree');
 
 CREATE TYPE public.review_response_status as ENUM ('Draft', 'Submitted');
 
+CREATE TYPE public.review_response_recommended_applicant_visibility as ENUM ('Original Response Visible to Applicant', 'Original Response Not Visible to Applicant');
```

Comment:
Why not just `true/false`? There's a lot of overlap in the wording of the field name and the values, so we can probably remove some redundancy?